EG23.tex

```diff
@@ -465,154 +440,117 @@ \subsection{Case 1: Greedy Separations}
+
+The accumulation of LWR used fuel in ``UNF Waiting for Reprocessing'' is induced
+by the limited constant reprocessing capacity for that fuel. The amount of UOX
+spent fuel in storage stops increasing when the production rate of used UOX fuel
+becomes smaller than its reprocessing rate, i.e. when the production of used
+fuel drops below $2\times83.3333$ tons.
  
-%As shown in the Figures \ref{fig:FC_Z} and \ref{fig:WR_Z}, the quantity of
-%plutonium in storage follows the variation in the amount of MOX1 fuel in
-%storage.
 
-%\begin{figure}[h!]
-%    \centering
-%    \subfigure[Fuel Resource in Storage (zoomed)\label{fig:FC_Z}]       {\epsfig{figure=img/FuelInStorage_1_zoom,width=0.48\textwidth}}
-%    \subfigure[Fuel waiting for reprocessing (zoomed)\label{fig:WR_Z}]  {\epsfig{figure=img/UNFWaitingReprocessing_1_zoom,width=0.48\textwidth}}
-%    \caption{Plutonium in storage.\label{fig:FC_WR_zoom} }
-%\end{figure}
+
+Before the start of the first FBR, the reprocessing capacities of the LWR fuel
+is higher than the demand on plutonium to fabricate the MOX fuel, creating an
+accumulation of plutonium prior to 2100, as shown in the Figure
+\ref{fig:FIS_1}. After 2100, with the decommissioning of the LWRs and the
+consequent expansion of SFRs, less LWRs used fuel is produced, allowing the
+reduction of the inventory of plutonium. The reprocessing capacities of the FBR
+used fuel are well adapted to the demand, but the plutonium storage experiences
+a small accumulation just before the SFRs switch to lower breeding ratio.
+ 
 
 \subsection{Case 2: On-demand Separations}
 
-The only differences in the results between the two cases can be seen in the
-behavior of the reprocessed fuel --- namely in the ``UNF fuel waiting for
-reprocessing'' and in the ``Fuel Resource in Storage''.
+The limitation added on the reprocessing capacities( Figure \ref{fig:ARR_2}) allows to limit the
+inventory of separated plutonium. Indeed, as seen in Figure \ref{fig:ARR_2}, the
+reprocessing follows more closely than in case 1 the plutonium need of the SFR
+fuel fabrication: the accumulation of plutonium is limited to $\sim500~t$
+(Figure \ref{fig:FIS_2}) versus $\sim1500~t$ in the first case. Nevertheless,
+one can still observe a large plutonium accumulation spike 2100 and 2145 that
+could be avoided with a better reprocessing capacity deployment.
 
 \begin{figure}[h!]
     \centering
-    \subfigure[Annual Reprocessing Rate\label{fig:ARR_2}]   {\epsfig{figure=img/AnnualReprocessingRate_2,width=0.48\textwidth}}
-    \subfigure[Fuel waiting for reprocessing]               {\epsfig{figure=img/UNFWaitingReprocessing_2,width=0.48\textwidth}}
-    \subfigure[Fuel Resource in Storage\label{fig:SFC_2}]  {\epsfig{figure=img/FuelInStorage_2,width=0.48\textwidth}}
-    \caption{Resources production and fuel loading.\label{fig:ARR_FWR_SFC_2} }
+    \subfigure[Case 1.\label{fig:ARR_1}] {\epsfig{figure=img/AnnualReprocessingRate_1,width=0.48\textwidth}}
```

Comment:
These plots occur twice... one of them should be "Fuel in Cooling Storage" or so...?